content/docs/user-guide/components/reference/animation/actor.md

```diff
@@ -1,35 +1,74 @@
++ **[Simple Motion](./simple-motion)** component - Use a single motion for your actor.
++ **[AnimGraph](./animgraph)** component - Use an animation graph to control your actor's motions.
 
 ## Actor Component Properties 
 
-![Actor component properties.](/images/user-guide/component/actor_component_properties.png)
+![Actor component properties.](/images/user-guide/components/reference/animation/actor-component.png)
 
-The **Actor** component has the following properties.
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Actor asset** | Sets the actor asset for this component. | Actor Asset | None |
+| **Attach To - Attachment type** | Sets the type of attachment to use when attaching to the target entity. | `None`, `Skin attachment` | `None` |
+| **Attach To - Target entity** | <p>Sets the entity to attach to.</p><p>*This field is available only if **Attachment type** is set to `Skin attachment`.*</p> | EntityId | None |
+| **Render options - Draw skeleton** | Draws the actor's joints. | Boolean | `Disabled` |
+| **Render options - Draw character** | Draws the actor's mesh. | Boolean | `Enabled` |
+| **Render options - Draw bounds** | Draws the actor's bounding box. | Boolean | `Disabled` |
+| **Render options - Skinning method** | Sets the skinning method to use for the actor. | `Dual quat skinning`, `Linear skinning` | `Dual quat skinning` |
+| **Out of view - Force update Joints** | Updates joint transforms when the actor is out of camera view. | Boolean | `Disabled` |
+| **Out of view - Bounding box configuration - Bounds type** | Sets the method used to compute the actor's bounding box. | `Static`, `Bone position-based`, `Mesh vertex-based` | `Static` |
+| **Out of view - Bounding box configuration - Automatically update bounds?** | <p>If `False`, actor bounds will only be calculated on activation or by calling a recalculation explicitly.  If `True`, actor bounds will be calculated at a regular interval determined by the **Update frequency** and **Update item skip factor** properties below.</p><p>*This field is available only if **Bounds type** is set to `Bone position-based` or `Mesh vertex-based`.*</p>| Boolean | `Enabled` |
+| **Out of view - Bounding box configuration - Update frequency** | <p>Sets the frequency of bounding box updates in hertz.</p><p>*This field is available only if **Bounds type** is set to `Bone position-based` or `Mesh vertex-based`.*</p> | 0.0 to Infinity | `0.0` |
+| **Out of view - Bounding box configuration - Update item skip factor** | <p>Sets bounding box updates to only calculate bounds based on every *n<sup>th</sup>* item (bone or vertex), where *n* is the **Update item skip factor**.</p><p>*This field is available only if **Bounds type** is set to `Bone position-based` or `Mesh vertex-based`.*</p> | 1 to Infinity | `1` |
+| **Out of view - Bounding box configuration - Expand by** | Expands the dimensions of a calculated bounding box by a percentage. | -99.999 to Infinity | `25.0` |
```

Comment:
I liked what you tried with the Camera component which had multiple possible configurations like this - I know that we can't embed tabs in tables, though. I'm fine with formatting as-is for now but it would be good to figure out how to handle a website experience where the UX in the editor is a dropdown/toggle that changes the UI (cc @yuyihsu @amzn-leenguy )